### PR TITLE
Adding recibe with complete JWT Auth

### PR DESCRIPTION
**Pull request by enBonnet**
This recipe includes:
- User signup
- User update
- User delete
- User login
- Valid JWT
- CRUD Products

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,6 +28,7 @@ Here you can find the most **delicious** recipes to cook delicious meals using o
 - [Heroku App](/heroku)
 - [GORM](/gorm)
 - [JWT](/jwt)
+- [User Auth whit JWT](/auth-jwt)
```
Comment (koddr) [inline README.md:31]: Typo: `whit` -> `with`

File: auth-jwt/config/config.go [hunk 2]
```diff
@@ -0,0 +1,18 @@
+package config
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/joho/godotenv"
```
Comment (koddr) [inline auth-jwt/config/config.go:7]: Why not built-in _lazy_ load to `.env` file?

File: README.md (hunk 1 above)
Comment (enBonnet) [reply README.md:31]: Cool! Thanks!

File: auth-jwt/config/config.go (hunk 2 above)
Comment (enBonnet) [reply auth-jwt/config/config.go:7]: How would you? I'm still learning golang and could help me to know...

**Comment by enBonnet** [comment]
> LGTM, but I wish to similar example without GORM (with plain SQL by [`sqlx`](https://github.com/jmoiron/sqlx), for example).

Would be great!

File: auth-jwt/config/config.go (hunk 2 above)
Comment (koddr) [reply auth-jwt/config/config.go:7]: I mean this one (from [joho/godotenv](https://github.com/joho/godotenv#usage)):

```go
import (
    // ...

    _ "github.com/joho/godotenv/autoload"
)

// ...

func main() {
    _ = godotenv.Load() // find and load .env life (by default)

   // ...
```

Comment (enBonnet) [reply auth-jwt/config/config.go:7]: Nice to know! Thanks!

**Pull request merged**